services/movie.py

```diff
@@ -0,0 +1,43 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+    genres_ids: list[int] | None = None,
+    actors_ids: list[int] | None = None
+) -> QuerySet:
+
+    queryset = Movie.objects.all()
+
+    if genres_ids:
+        queryset = queryset.filter(genres__in=genres_ids)
+
+    if actors_ids:
+        queryset = queryset.filter(actors__in=actors_ids)
+
+    return queryset
+
+
+def get_movie_by_id(
+    movie_id: int | None = None
+) -> Movie:
```

Comment:
The `get_movie_by_id` function uses `.get()`, which will raise a `Movie.DoesNotExist` exception if no movie with the given ID is found. Consider handling this exception or updating the return type annotation to indicate that an exception might be raised.